src/scripts/party/Party.ts

```diff
@@ -178,6 +178,27 @@ class Party implements Feature {
         return Math.min(1, Math.max(0.2, 0.1 + (highestRegion / 10)));
     }
 
+    public gainEffortPoints(pokemon: PartyPokemon, shiny: boolean, number = GameConstants.BASE_EP_YIELD): number {
+        let EPNum = number;
+
+        if (pokemon.pokerus < GameConstants.Pokerus.Contagious) {
+            return 0;
+        }
+
+        if (App.game.multiplier.getBonus('ev') > 1) {
```

Comment:
I don't think this is needed, but i am not sure. It does not make sense that it can be below one, and if it can, it's because we have added a weird debuff, and then it won't work anyways.